libid/fractalp.cpp

```diff
@@ -2303,6 +2303,16 @@ FractalSpecific g_fractal_specific[] =
         mandelbrot_mix4_fp_fractal, mandelbrot_mix4_fp_per_pixel, mandelbrot_mix4_setup, standard_fractal,
         STD_BAIL_OUT
     },
+    {
+        "burningship",
```

Comment:
Please use `burning-ship` instead of `wordssmashedtogether`

Reply:
Okay